client/src/components/MeetingExperience.tsx

```diff
@@ -140,19 +137,15 @@ const _createCustomAdapter = async (
     }
   );
 
-  const threadId = getChatThreadIdFromTeamsLink(locator.meetingLink);
-
-  const chatClient = chatEnabled
-    ? createStatefulChatClient({
-        userId,
-        displayName,
-        endpoint,
-        credential
-      })
-    : createStubChatClient(userId, threadId);
```

Comment:
This `chatEnabled` conditional is a recent change that we should keep: https://github.com/Azure-Samples/communication-services-virtual-visits-js/pull/43/files#diff-98e77a3eb2de1327d4cdf34141837b5ca45fe5dad35a2447a6e0ec3eafb272b6